src/Scene/NodeProcess.js

```diff
@@ -253,48 +237,36 @@ function updateNodeImagery(scene, quadtree, node, layersConfig, force) {
             }
         }
 
-        let ancestor = null;
-
+        node.layerUpdateState[layer.id].newTry();
+        const searchInParent = !node.isColorLayerLoaded(layer.id) && node.parent.isColorLayerLoaded(layer.id);
         const currentLevel = node.materials[RendererConstant.FINAL].getColorLayerLevelById(layer.id);
-        // if this tile has no texture (level == -1), try use one from an ancestor
-        if (currentLevel === -1) {
-            ancestor = findAncestorWithValidTextureForLayer(node, l_COLOR, layer);
-        } else {
+
+        if (currentLevel > -1) {
             var targetLevel = chooseNextLevelToFetch(layer.updateStrategy.type, node.level, currentLevel, layer.updateStrategy.options);
-            if (targetLevel === currentLevel) {
+            if (targetLevel <= currentLevel) {
                 continue;
             }
-            if (targetLevel < node.level) {
-                ancestor = node.getNodeAtLevel(targetLevel);
-            }
         }
 
-        node.layerUpdateState[layer.id].newTry();
-
         const command = {
             /* mandatory */
             layer,
             requester: node,
             priority: nodeCommandQueuePriorityFunction(node),
             earlyDropFunction: refinementCommandCancellationFn,
-            /* specific params */
-            ancestor,
+            parentTextures: searchInParent ? node.parent.getLayerTextures(l_COLOR, layer.id) : undefined,
```

Comment:
Yes it is true, it should be seen in a new PR

We must not lose sight of the prioritization of the requests with the scheduler.